addon/components/horizontal-list-swiper/gesture/component.js

```diff
@@ -6,22 +6,34 @@ import Ember from 'ember';
 import RecognizerMixin from 'ember-gestures/mixins/recognizers';
 import layout from './template';
 import { run } from '@ember/runloop';
+import {getOwner} from '@ember/application';
 
 const {
   String
 } = Ember;
 
 export default Component.extend(RecognizerMixin, {
-
+  window: window,
```

Comment:
Should use the window service if open sourced

Reply:
yes added a bullet point - need to refresh https://github.com/britishgas-engineering/ember-window or have an addon sharing usual services (viewport too for example). Can it be another PR pls